axelrod/tests/strategies/test_grudger.py

```diff
@@ -236,22 +259,105 @@ class TestGeneralSoftGrudger(TestPlayer):
     }
 
     def test_strategy(self):
-        """Test strategy with multiple initial parameters"""
-
-        # Starts by cooperating.
-        self.first_play_test(C)
-
-        # Testing default parameters of n=1, d=4, c=2 (same as Soft Grudger)
-        actions = [(C, D), (D, D), (D, C), (D, C), (D, D), (C, D), (C, C), (C, C)]
-        self.versus_test(axelrod.MockPlayer(actions=[D, D, C, C]), expected_actions=actions)
-
-        # Testing n=2, d=4, c=2
-        actions = [(C, D), (C, D), (D, C), (D, C), (D, D), (D, D), (C, C), (C, C)]
-        self.versus_test(axelrod.MockPlayer(actions=[D, D, C, C]), expected_actions=actions,
-                         init_kwargs={"n": 2})
-
-        # Testing n=1, d=1, c=1
-        actions = [(C, D), (D, D), (C, C), (C, C), (C, D), (D, D), (C, C), (C, C)]
-        self.versus_test(axelrod.MockPlayer(actions=[D, D, C, C]), expected_actions=actions,
-                         init_kwargs={"n": 1, "d": 1, "c": 1})
-
+        # the default follows SoftGrudger
+        super(TestGeneralSoftGrudger, self).test_strategy()
+
```

Comment:
I am a bit weary of PRs that change things that have been (very recently) added when the motivation for change seems stylistic. This takes a significant cognitive effort by the reviewers to make sure we're not breaking anything. If you recall there were a fair few times I had to explain why something was done a particular way in the fingerprint class. 

Please keep these as they are. #884 is a simple issue: refactor tests to use `versus_test`. If you catch a case that is missing by all means add it.

> whether or not these are good reasons, I do think it useful to have this inherit from TestSoftGrudger which will test the default behavior and reset

`test_reset` can actually be removed entirely by the way, this is now being tested by the parent class `TestPlayer`.